### PR TITLE
feat and styling

### PR DESCRIPTION
**Pull request by shlok30**
Added buttons and alerts and fixed some identations

Please review the following components 

Alerts - https://water-style-ui.netlify.app/components/alert.html
Buttons - https://water-style-ui.netlify.app/components/button.html

Thanks in advance

### CODE REVIEW

File: Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.alert {
+    color: var(--white-color);
+    width: 100%;
```
Comment (mohitdhatrak) [inline Components/alert.css]: don't use color name itself in variable, give generic names like text-color, dark-bg-text-color etc. Else it kills the purpose of variables

File: Components/button.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 5px;
+  }
+  
+  .floating-btn {
+    height: 2.5rem;
+    width: 2.5rem;
+    border-radius: 999px;
```
Comment (mohitdhatrak) [inline Components/button.css]: don't use absolute units for such high values, use rem. If there's a reason let me know i may be wrong. For circle u can use border-radius 50%

**Review by mohitdhatrak — COMMENTED** [review]
Small suggestions, check them. Looks good otherwise!

File: Components/button.css (hunk 2 above)
Comment (syed-shaida-hussain) [inline Components/button.css]: You can simply use border-radius: 50% ; and it will still give you a circle shaped button

**Review by syed-shaida-hussain — APPROVED** [review]
Try to change absolute units to relative units. Rest looks good.

**Review by ark2002 — APPROVED** [review]
apart from the changes suggested everything looks good

File: Components/alert.css (hunk 1 above)
Comment (shlok30) [reply Components/alert.css]: Thanks for the suggestion

File: Components/button.css (hunk 2 above)
Comment (shlok30) [reply Components/button.css]: I saw 999px in the figma file that was shared and hence thought I'll keep it that way

Comment (shlok30) [reply Components/button.css]: Understood, thanks

**Pull request merged**